README.md

```diff
@@ -231,6 +233,18 @@ dev-ops/common/actions/test.sh
 echo __TEST__
 ```
 
+#### Require
+
+It may be necessary to require a placeholder to be set, but can't set right away. One such example might be a system dependent path. PSH allows you to communicate this to the user by adding this:
+
+```xml
+<placeholder>
+    <require name="FOO" description="Foo needs to be a reference to bar"/>
+</placeholder>
+```
+
+Now unless foo is set, it is not possible to execute any psh script. The description is optional and can be omitted.
```

Comment:
```suggestion
Although `FOO` is now set, it is not possible to execute any psh script. The description is optional and can be omitted.
```

Reply:
This reverses the meaning, doesn't it?

Reply:
Sorry, you're right. I misunderstood the sentence at first.